motioneye/mediafiles.py

```diff
@@ -742,6 +756,9 @@ def poll_movie_process(pictures):
                 else:
                     raise
 
+            if output is None:
+                output = ''
+
```

Comment:
```suggestion
            output = output or ''
```
All falsey values (False, 0, (), {}, [], etc.) are just as risky as None.